### PR TITLE
[skip ci] Fix header uniformity for Import, Mesh, Points, Raytracing

### PR DESCRIPTION
**Pull request by luzpaz**

### CODE REVIEW

File: src/Mod/Mesh/BuildRegularGeoms.py [hunk 1]
```diff
@@ -1,10 +1,11 @@
+# Copyright (c) 2005 Berthold Grupp
```
Comment (vocx-fc) [inline src/Mod/Mesh/BuildRegularGeoms.py]: This file starts with a Python docstring already, that is, triple quotes. This is probably the best way to do it. The author and license should be provided after this documentation block.
```py
"""Python Module for building solid regular geometric objects.
...
"""
# Copyright (c) 2005 Berthold Grupp
# License: LGPL
```

Also, the full license block should be placed explicitly like in other source files.

File: src/Mod/Raytracing/App/resources/FCAnimation.pov [hunk 2]
```diff
@@ -87,7 +87,7 @@
 };
 
 
-// Textur f�r Koordinatenlinien
+// Textur f�r Koordinatenlinien
```
Comment (vocx-fc) [inline src/Mod/Raytracing/App/resources/FCAnimation.pov]: // Texture for coordinate lines

File: src/Mod/Raytracing/App/resources/FCAnimation.pov [hunk 3]
```diff
@@ -98,7 +98,7 @@
     finish {LineFinish}
   };
   /* 
-  // H�henlinien
+  // H�henlinien
```
Comment (vocx-fc) [inline src/Mod/Raytracing/App/resources/FCAnimation.pov]: // High lines

File: src/Mod/Raytracing/App/resources/FCAnimation.pov [hunk 4]
```diff
@@ -191,7 +191,7 @@ plane {               // checkered floor
 }
 
 
-// T�re
+// T�re
```
Comment (vocx-fc) [inline src/Mod/Raytracing/App/resources/FCAnimation.pov]: // Doors

**Comment by wwmayer** [comment]
c766d7579

File: src/Mod/Mesh/BuildRegularGeoms.py (hunk 1 above)
Comment (luzpaz) [reply src/Mod/Mesh/BuildRegularGeoms.py]: Will fix this in a future commit

**Pull request merged**